src/libs/kata-types/src/mount.rs

```diff
@@ -23,6 +23,10 @@ pub const KATA_SHAREDFS_GUEST_PREMOUNT_TAG: &str = "kataShared";
 /// KATA_EPHEMERAL_VOLUME_TYPE creates a tmpfs backed volume for sharing files between containers.
 pub const KATA_EPHEMERAL_VOLUME_TYPE: &str = "ephemeral";
 
+/// KATA_K8S_LOCAL_STORAGE_TYPE is used for k8s empty dir (a disk-backed volume),
+/// to create a local directory inside the VM for sharing files between containers.
+pub const KATA_K8S_LOCAL_STORAGE_TYPE: &str = "local";
+
 /// KATA_HOST_DIR_TYPE use for host empty dir
 pub const KATA_HOST_DIR_VOLUME_TYPE: &str = "kata:hostdir";
```

Comment:
Will the KATA_HOST_DIR_VOLUME_TYPE be needed once we use KATA_K8S_LOCAL_STORAGE_TYPE replaced KATA_HOST_DIR_VOLUME_TYPE?

Reply:
Thx bro @lifupan Sure, don't aggree more with it.